docs/source/settings.rst

```diff
@@ -459,6 +459,22 @@ Set to ``*`` to disable checking of Front-end IPs (useful for setups
 where you don't know in advance the IP address of Front-end, but
 you still trust the environment)
 
+server_name
+~~~~~~~~~~~~~~~~~~~
+
+* ``--server_name STRING``
```

Comment:
Here and below, is what I am referring to when I talk about hyphenation.

```
``--server-name STRING``
```

No?